packages/ui-react/src/containers/AccountModal/forms/ChooseOffer.tsx

```diff
@@ -40,15 +44,38 @@ const ChooseOffer = () => {
 
       if (!offer) return;
 
-      await chooseOffer.mutateAsync(offer);
+      const baseUrl = window.location.origin;
+
+      const url = await chooseOffer.mutateAsync({
```

Comment:
This will break current implementations. It is also used for update subscription and selecting an offer from Cleen implementation, should not be invoked if access bridge is not present.